seungjong/app.js

```diff
@@ -112,6 +112,68 @@ app.get('/users/:userId/posts', async (req, res, next) => {
 });
 
 
+app.patch('/users/:userId/:postId', async (req, res, next) => {
+    const { userId, postId } = req.params;
+    const { userName, title, content } = req.body;
+
+    await dataSource.query(`    
+        UPDATE posts
+        SET
+        content = ?
+        WHERE id = ?     
+        `, [content, postId]
+        )
+    
+    const result = await dataSource.query(`
+        SELECT
+        u.id,
+        u.name,
+        p.id,
+        p.title,
+        p.content
+        FROM users as u
+        JOIN posts as p
+        ON u.id = p.user_id
+        WHERE p.id = ?;
+    `, [postId]);
+
+    res.status(200).json({ data: {
+        userId : userId,
+        userName: result[0].name,
+        postingId : postId,
+        postingTitle: result[0].title,
+        postingContent: content
+    } });
```

Comment:
response로 보내주는 result는 실제로 업데이트가 이뤄지고 난 다음의 값들로 보내질 수 있도록 해주세요.
```suggestion
    res.status(200).json({ data: result );
```